src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -89,22 +90,21 @@ Server: kong/1.2.1
 
 This is expected, as Kong does not yet know how to proxy the request.
 
-## Set up an echo-server
+## Set up an echo service
 
-Set up an echo-server application to demonstrate how
-to use the {{site.kic_product_name}}:
+Set up an echo service to demonstrate how to use the {{site.kic_product_name}}:
 
 ```bash
-$ kubectl apply -f https://bit.ly/echo-service
+$ kubectl apply -f {{site.links.web}}/kubernetes-ingress-controller/{{page.kong_version}}/examples/echo-service.yaml
 ```
 
-## Add a Gateway class and gateway
+## Add a `GatewayClass` and `Gateway`
```

Comment:
```suggestion
## Add a GatewayClass and Gateway
```
Just changed these to plain text to fit our style guidelines about headers: https://docs.konghq.com/contributing/markdown-rules/#headers

I think we opted for plain text only in headers because the code made it harder to read.